commodore/component-template/{{ cookiecutter.slug }}/.github/changelog-configuration.json

```diff
@@ -0,0 +1,30 @@
+{
+    "pr_template": "- ${{TITLE}} (#${{NUMBER}})",
```

Comment:
Seems like this trips up the jinja template of cookiecutter... You might have to escape the `{{` and `}}`

Reply:
can I test it out locally?

Reply:
I just `{% raw %}`d the whole file :)